docs/docs/guides/evaluation/guardrails_and_monitors.md

```diff
@@ -5,139 +5,482 @@ import TabItem from '@theme/TabItem';
+## Introduction
 
-## Core Concepts
+Building production LLM applications? Two questions likely keep you up at night:
+1. How do you ensure your LLMs generate safe, appropriate content?
+2. How do you measure and improve output quality over time?
 
-The foundation of Weave's evaluation system is the `Scorer` class. This abstract base class defines a scoring interface through its `score` method, which concrete implementations use to provide specific evaluation metrics. For a comprehensive overview of available metrics and custom scorer implementation, see [Evaluation Metrics](./scorers.md).
+Weave's unified scoring system answers both questions through a simple yet powerful framework. Whether you need active safety controls (guardrails) or passive quality monitoring, this guide will show you how to implement robust evaluation systems for your LLM applications.
 
-Here's a basic example of a custom scorer:
+## Overview
+
+The foundation of Weave's evaluation system is the [**Scorer**](./scorers.md) - a component that evaluates your function's inputs and outputs to measure quality, safety, or any other metric you care about. Scorers are versatile and can be used in two ways:
+
+- **As Guardrails**: Block or modify unsafe content before it reaches users
+- **As Monitors**: Track quality metrics over time to identify trends and improvements
+
+:::note Terminology
+Throughout this guide, we'll refer to functions decorated with `@weave.op` as "ops". These are regular Python functions that have been enhanced with Weave's tracking capabilities.
+:::
+
+:::tip Ready-to-Use Scorers
+While this guide shows you how to create custom scorers, Weave comes with a variety of [predefined scorers](./scorers.md#predefined-scorers) that you can use right away, including:
+- Toxicity detection
+- Prompt injection detection
+- Response quality evaluation
+- Factual consistency checking
+- And more!
+
+Check out our [predefined scorers list](./scorers.md#predefined-scorers) to get started quickly with production-ready evaluation.
+:::
+
+### Guardrails vs. Monitors: When to Use Each
+
+While scorers power both guardrails and monitors, they serve different purposes:
+
+| Aspect | Guardrails | Monitors |
+|--------|------------|----------|
+| **Purpose** | Active intervention to prevent issues | Passive observation for analysis |
+| **Timing** | Real-time, before output reaches users | Can be asynchronous or batched |
+| **Performance** | Must be fast (affects response time) | Can be slower, run in background |
+| **Sampling** | Usually every request | Often sampled (e.g., 10% of calls) |
+| **Control Flow** | Can block/modify outputs | No impact on application flow |
+| **Resource Usage** | Must be efficient | Can use more resources if needed |
+
+For example, a toxicity scorer could be used to:
```

Comment:
Remove "to" , it doesn't match with "As" in bullets

Reply:
done